src/components/settings-components/SettingsView.vue

```diff
@@ -388,6 +380,14 @@ import CdnProvider from '../../providers/generic/connection/CdnProvider';
                         },
                         'fa-folder-open',
                         () => this.emitInvoke('ChangeSteamDirectory')
+                    ),
+                    new SettingsRow(
+                        'Debugging',
+                        'Validate Steam game installation',
```

Comment:
`Validate` is probably the wrong term? Makes it sound like it would just start Steam's validate functionality which we do, but it's not the only thing.

Options:
- `Clean and validate <game> installation`
- `Reset <game> installation`
- `Something else?`

Would be good to have the game name dynamic, similar to how it is for the `Change <game> folder` setting. It makes it easier to ensure that people don't click it for the wrong game.

Reply:
I went with `Reset <game> installation` as I find "reset" suggests kind of "factory reset" type of thing which is what we do here.